radicle-node/Cargo.toml

```diff
@@ -36,6 +36,11 @@ snapbox = { version = "0.4.3", optional = true }
 tempfile = { version = "3.3.0" }
 thiserror = { version = "1" }
 
+[dependencies.git2]
+version = "0.16.1"
+default-features = false
+features = ["vendored-libgit2"]
```

Comment:
I think you can access this via `radicle::git::raw` if I remember. Better than re-importing due to the extra working when updating versions.

Reply:
Actually, this can be removed because it was for the `commit` function. Thanks for the tip, regardless!